packages/msedit/build.sh

```diff
@@ -0,0 +1,29 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/microsoft/edit"
+TERMUX_PKG_DESCRIPTION="A simple editor for simple needs (Microsoft Edit)"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="1.2.0"
+TERMUX_PKG_SRCURL="https://github.com/microsoft/edit/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz"
+TERMUX_PKG_SHA256=e4ba6ff1bfecfeff2492306f5850c714bf50ffdb3cc3bb5be3aa987289f240fe
+TERMUX_PKG_DEPENDS="libicu"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_AUTO_UPDATE=true
+
+termux_step_pre_configure() {
+	# Do not pin the compiler version to nightly.
+	rm "$TERMUX_PKG_SRCDIR/rust-toolchain.toml"
+	termux_setup_rust
+}
+
+termux_step_make() {
+	# Allow nightly features
+	export RUSTC_BOOTSTRAP=1
+	cargo build --release
+}
+
+termux_step_make_install() {
+	install -Dm755 "$TERMUX_PKG_SRCDIR/target/release/edit" "$TERMUX_PREFIX/bin/msedit"
+	ln -sf "./msedit" "$TERMUX_PREFIX/bin/edit" # and symlink bin/edit
+	install -Dm644 "$TERMUX_PKG_SRCDIR/assets/manpage/edit.1" "$TERMUX_PREFIX/share/man/man1/msedit.1"
+	ln -sf "./msedit.1.gz" "$TERMUX_PREFIX/share/man/man1/edit.1.gz" # and symlink man1/edit.1
+}
```

Comment:
I think making the symlinks relative makes more sense here.
The man page symlink additionally needs a `.gz` at the end since the manpages are compressed as part of wrapping up the package.